lib/usb_serial.dart

```diff
@@ -335,12 +341,12 @@ class UsbDevice {
     if (!(other is UsbDevice)) {
       return false;
     }
-    return _Equality.deepEq(_props, other._props);
+    return this.deviceName == other.deviceName;
```

Comment:
deviceName is device path in usbfs so it 1) not null 2) is unique.
I think it could be used for both equals() and hashCode().